### DISCUSSION TITLE
does pino 8 work with fastify 3?

### DISCUSSION THREAD

**Discussion by DanielRuf** [fastify/fastify#5013] [General] [answered]
At https://github.com/pinojs/pino/pull/1249 it says it it a preparation for fastify 4.

But does pino 8 also work with fastify 3? Because the fastify 4 upgrade will take more effort for us so we only want to upgrade pino at the moment.

**Comment by mcollina** [comment, marked as answer]
should be compatible.

Having said that, fastify v3 is out of LTS.

**Reply by DanielRuf** [reply]
Perfect, thanks.

Well, it takes some time to upgrade a legacy application and there have to be new contracts created / negotiated with the customer when we do the bigger changes (I have no power regarding this decision). And the biggest change so far would be to migrate from fastify v3 to v4.

The upgrade is definitely planned, but not for now.

**Reply by Eomm** [reply]
> The upgrade is definitely planned, but not for now.

@DanielRuf If you need help, [NearForm](https://www.nearform.com/) is a black belt on these tasks - feel free to DM me (see my profile for your preferred platform).
